### PR TITLE
38797 return pi data

### PR DESCRIPTION
**Pull request by tpharrison**
## Original issue(s)
Update personal_informations_controller#show to return preferred name and gender identity. 

- `preferred_name` and `gender_identity` fields are retrieved from VA Profile demographics API through `VAProfile::Demographics::Service`.
- This controller method already returns user information - `id`, `type`, `gender`, and `birth_date`. 
- Feature flag `profile_show_demographics` used to toggle API call to VAProfile to retrieve this data.
- Swagger docs and specs have been updated.

Associated ticket is [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38797).

### CODE REVIEW

File: spec/serializers/personal_information_serializer_spec.rb [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
 
 require 'rails_helper'
+require 'va_profile/demographics/demographic_response'
 
 RSpec.describe PersonalInformationSerializer do
-  subject { serialize(mvi_profile, serializer_class: described_class) }
-
-  let(:mvi_profile) { MPIData.for_user(create(:user, :loa3)).profile }
-  let(:attributes) { JSON.parse(subject)['data']['attributes'] }
+  before do
```
Comment (RachalCassity) [inline spec/serializers/personal_information_serializer_spec.rb:7]: Is there a reason why let is not being used to define variables?

Comment (RachalCassity) [reply spec/serializers/personal_information_serializer_spec.rb:7]:
```
  let(:demographics) { get_demographics }
  let(:response) { serialize(demographics, serializer_class: described_class) }
  let(:attributes) { JSON.parse(response)['data']['attributes'] }
```

Comment (tpharrison) [reply spec/serializers/personal_information_serializer_spec.rb:7]: HI @RachalCassity - no, I can make that change.

Comment (tpharrison) [reply spec/serializers/personal_information_serializer_spec.rb]: Thanks @RachalCassity!

**Comment by tpharrison** [comment]
Hi @RachalCassity - sorry to bother you with this, but I keep getting [this workflow run failure](https://github.com/department-of-veterans-affairs/vets-api/actions/runs/2191538230) when trying to merge this PR.

I've had workflow issues in the past, and just kept re-running until it passed.  

Any suggestions on how to fix this?  Thanks!

**Comment by RachalCassity** [comment]
Looking now.

**Comment by RachalCassity** [comment]
Have you pulling from master and seeing if there are any merge conflicts?

**Comment by tpharrison** [comment]
It looks like it was deployed.  Is everything good?

If not, I just did a `git pull origin master` and merged locally without any issues.  should I restore the branch and push?

**Pull request merged**